contracts/src/libraries/FROST.sol

```diff
@@ -3,54 +3,123 @@ pragma solidity ^0.8.30;
 
 import {Secp256k1} from "@/libraries/Secp256k1.sol";
 
-/// @title FROST
-/// @notice Implementation of FROST(secp256k1, SHA-256) cyphersuite.
+/**
+ * @title FROST
+ * @notice Implementation of the FROST(secp256k1, SHA-256) ciphersuite.
+ */
 library FROST {
     using Secp256k1 for Secp256k1.Point;
 
+    // ============================================================
+    // TYPES AND STRUCTS
+    // ============================================================
+
+    /**
+     * @notice Type alias for a FROST participant identifier.
```

Comment:
When do we call something "identifier" and when do we call it "id"?

Reply:
This will be handled in either the next PR or the Glossary PR. Right now, I am not sure.